src/data/int/basic.lean

```diff
@@ -159,6 +159,12 @@ lemma coe_nat_succ_pos (n : ℕ) : 0 < (n.succ : ℤ) := int.coe_nat_pos.2 (succ
 @[simp, norm_cast] theorem coe_nat_abs (n : ℕ) : |(n : ℤ)| = n :=
 abs_of_nonneg (coe_nat_nonneg n)
 
+lemma add_abs_nonneg (a : ℤ) : 0 ≤ a + |a| :=
```

Comment:
Can you move this to be next to `neg_le_abs_self`?

Reply:
moved it next to `neg_abs_le_self` instead because the proof requires `[covariant_class α α (+) (≤)]`.